src/app/components/combo-box.tsx

```diff
@@ -0,0 +1,69 @@
+"use client"
+import { DropdownItem } from "../models/dropdown-item"
+import { useState } from "react"
```

Comment:
It would be nice if you can rearrange the imports as per the below order
1. React imports
2. Third party imports
3. Internal imports